components/Blog/BlogCard/__tests__/index.test.tsx

```diff
@@ -0,0 +1,39 @@
+import { render } from '@testing-library/react';
+import { LocaleProvider } from '../../../../providers/localeProvider';
+import englishMessages from '../../../../i18n/locales/en.json';
+import BlogCard from '..';
+
+jest.mock('next/router', () => ({
+  useRouter: jest.fn().mockReturnValue({}),
+}));
+
+describe('BlogCard component', () => {
+  it('renders correctly', () => {
+    const { container } = render(
+      <LocaleProvider
```

Comment:
You should not provide the LocaleProvider here. Just se how other tests did testing here for i18n.

The tl;dr is just import the IntlProvider from react-intl, but without actually passing any data, just the onError callback. (See other tests)

The reasoning behind is that we don't want to have translations (even sources) of any kind on our unit tests as copy-write tends to change over time...

Reply:
I tried using `IntlProvider` in the unit tests before but it breaks because `LocalizedLink` uses LocaleProvider. What I can do is mock LocaleProvider, which might solve the issue, so should I rather mock it?

Reply:
I've changed it to mocking `useLocale`. Let me know if that's sufficient.